GoogleUtilities/Environment/third_party/GULAppEnvironmentUtil.m

```diff
@@ -272,7 +272,11 @@ + (NSString *)applePlatform {
 #if TARGET_OS_MACCATALYST
   applePlatform = @"maccatalyst";
 #elif TARGET_OS_IOS
-  applePlatform = @"ios";
+  if (@available(iOS 14.0, *)) {
```

Comment:
Should we also add Catalyst at the same time?

Reply:
Taking into account how ugly are the availability checks I would prefer to keep using `#if TARGET_OS_MACCATALYST` that works just fine.

Reply:
We have to use the runtime check for iOS on mac because at compile time it is still iOS.